internal/services/applicationinsights/sdk/2022-04-01/applicationinsights/client.go

```diff
@@ -0,0 +1,18 @@
+package applicationinsights
```

Comment:
rather than embedding this can we update this to use `go-azure-sdk` directly? https://github.com/hashicorp/go-azure-sdk/tree/main/resource-manager/applicationinsights

Reply:
Hi @tombuildsstuff, thank you for your reminding. The SDK has been updated. Kindly help review it. Thanks.
BTW, this PR is a renew one for [this](https://github.com/hashicorp/terraform-provider-azurerm/pull/17161) reverted merged PR.